src/styles.scss

```diff
@@ -1003,13 +1003,7 @@ body {
     }
   }
 }
-/* Reviews ---------------------(end) */
 
-/* Questions ---------------------(start) */
-
-/* Questions ---------------------(end) */
-
-/* Shared ---------------------(start) */
 .starInputContainer {
```

Comment:
Did you mean to delete the Shared marker?

Reply:
No, this was a merge conflict tat I just accepted incoming changes. Not intentional sorry!